docs/Tutorial_Matrix_Profile_Top_Ten_short.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>Let's start off by showing <code>A</code> and <code>B</code> (side-by-side) and asking the  reader, "are A and B the same? are they similar?  how would you quantify their similarity?" Then plot <code>A</code> and <code>B</code> on top of each other to show that they are actually different lengths but that <code>B</code> is stretched (or <code>A</code>  is compressed)  relative to the other. In other words, in this case, <code>A</code> and <code>B</code> have very similar shapes but, due to the stretching/compressing, they are said to be "out-of-phase" with each other. If you compute the Euclidean distance between <code>A</code> and <code>B</code> as is, then they'd have a large Euclidean distance between them. However, if you could figure out the best way to stretch/compress <code>A</code> or <code>B</code> so as to best match the locations of the peaks/valleys (i.e., bring them "in-phase"), then the Euclidean distance between would become smaller. One obvious way to accomplish this is by uniformly scaling (i.e., stretching/compressing) <code>A</code> or <code>B</code> by some constant scaling factor...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I'll adjust the notebook according to this guide.</p><p> </p>